index.js

```diff
@@ -182,24 +189,44 @@ Liftoff.prototype.launch = function (opts, fn) {
     return this.completions(completion);
   }
 
+  opts = mapValues(opts);
+
+  var getEnvironment;
+  if (process.env.__FLAGGED_RESPAWNED) {
+    opts.argv = process.argv;
+    getEnvironment = function(opts) {
```

Comment:
Can we change the implementation to avoid this?

Reply:
The `argv` is being built inside `buildEnvironment`, so can't we go back to just doing it inside the `flaggedRespawn` callback?

Reply:
When respawned, `buildEnvironment` and given `fn` function is not execute in callback.
(See [js-liftoff/index.js#L197](https://github.com/js-cli/js-liftoff/blob/master/index.js#L197) and [js-flagged-respawn/index.js#L14](https://github.com/js-cli/js-flagged-respawn/blob/master/index.js#L14)).